src/main/java/de/pi3g/pi/oled/RotatableOLEDDisplay.java

```diff
@@ -0,0 +1,101 @@
+package de.pi3g.pi.oled;
+
+import com.pi4j.io.i2c.I2CFactory;
+
+import java.io.IOException;
+
+public class RotatableOLEDDisplay extends OLEDDisplay {
+
+    private byte ROTATION = 0;
+
+    public RotatableOLEDDisplay() throws IOException, I2CFactory.UnsupportedBusNumberException {
+    }
+
+    public RotatableOLEDDisplay(int displayAddress) throws IOException, I2CFactory.UnsupportedBusNumberException {
+        super(displayAddress);
+    }
+
+    public RotatableOLEDDisplay(int busNumber, int displayAddress) throws IOException, I2CFactory.UnsupportedBusNumberException {
+        super(busNumber, displayAddress);
+    }
+
+    public int getWidth() {
+        switch (ROTATION) {
+            //Rotated by 90° or 270°
+            case 1:
+            case 3:
+                return super.getHeight();
+            //Rotated by 0° or 180°
+            case 0:
+            case 2:
+            default:
+                return super.getWidth();
+        }
+    }
+
+    public int getHeight() {
+        switch (ROTATION) {
+            //Rotated by 90° or 270°
+            case 1:
+            case 3:
+                return super.getWidth();
+            //Rotated by 0° or 180°
+            case 0:
+            case 2:
+            default:
+                return super.getHeight();
+        }
+    }
+
+
+    /**
+     * Set the rotation of the display (so that it can be mounted upright or can be flipped).
+     *
+     * @param angle Orientation angle of the display in degrees (clockwise).<br>
+     *              Allowed values are only 0°, 90°, 180° and 270°
+     */
+    public void setRotation(int angle) {
```

Comment:
I would prefer to pass an enum value here, to make it transparent to the user at compile time what values are possible. Values could be something like:
* `DEG_0` (which would be the default)
* `DEG_90`
* `DEG_180`
* `DEG_270`